src/components/icons.js

```diff
@@ -47,7 +48,10 @@ export const icon = (shape, { size = 16, ...props } = {}) => {
   return _.invokeArgs(shape, [{ size, 'data-icon': shape, ...props }], iconDict)
 }
 
-export const spinner = props => icon('loadingSpinner', _.merge({ size: 24, style: { color: colors.primary() } }, props))
+export const spinner = ({ message = 'Loading', ...props }) => h(Fragment, [
+  icon('loadingSpinner', _.merge({ size: 24, style: { color: colors.primary() } }, props)),
+  h(DelayedRender, { delay: 150 }, [div({ className: 'sr-only', role: 'alert' }, [message])])
```

Comment:
@msilva-broad This type of approach might work better on the spend report instead of the aria-live regions.